### PR TITLE
main menu style in process

### PR DESCRIPTION
**Pull request by Rightnoe**
Main menu styles, issue item needs fix.

### CODE REVIEW

File: src/scss/components/c-language-switcher.scss [hunk 1]
```diff
@@ -3,7 +3,8 @@
     -webkit-appearance: none;
        -moz-appearance: none;
             appearance: none;
-    padding-right: 25px;
+    padding-right: 5px;
```
Comment (clementprdhomme) [inline src/scss/components/c-language-switcher.scss]: These changes break the language switcher in the footer. What did you try to do?

Language selector with the text overlapping the arrow

File: src/scss/components/c-issue-card.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.c-issue-card {
```
Comment (clementprdhomme) [inline src/scss/components/c-issue-card.scss:1]: You don't use this style anywhere. Did you forget to push something else?

File: src/scss/style.scss [hunk 3]
```diff
@@ -541,7 +541,7 @@ a:hover, a:active {
 
 .main-navigation ul li:hover > ul,
 .main-navigation ul li.focus > ul {
-  left: auto;
+  left: -70%;
```
Comment (clementprdhomme) [inline src/scss/style.scss]: There seem to be some issues positioning the menu:
Sub-menu not aligned with the item
and the border should be the length of the text, I guess:
Border longer than the text

**Review by clementprdhomme — CHANGES_REQUESTED** [review]
Hey @Rightnoe,

Can you detail what this PR is about? I saw that you modify several files that don't have anything to do with the description.

I also put a few comments in the code.

File: src/scss/components/c-language-switcher.scss (hunk 1 above)
Comment (Rightnoe) [reply src/scss/components/c-language-switcher.scss]: please ignore, I just wanted to upload the main menu

File: src/scss/components/c-issue-card.scss (hunk 2 above)
Comment (Rightnoe) [reply src/scss/components/c-issue-card.scss:1]: I created it for "issue" index style, you can ignore it too

File: src/scss/style.scss (hunk 3 above)
Comment (Rightnoe) [reply src/scss/style.scss]: Thanks! i'll fix it

**Pull request merged**